views/index.ejs

```diff
@@ -33,8 +33,13 @@
       (k,v) => k==='server' ? undefined : v)};` %>
   </script>
     <div id="scene_container" class="map_container"></div>
-
-    <div id="panels"></div>
+    <div class="side_panel__container">
```

Comment:
This mark-up comes directly from the old `app_panel.dot`. I didn't want to risk breaking things so I didn't change it but it could be simplified later. The ideal target would be to have a single `react-root` node, or something like that.